SADL/AM-CDM-part.sadl

```diff
@@ -0,0 +1,27 @@
+uri "http://kdl.ge.com/AM-CDM-part" alias AM-CDM-part version "$Revision:$ Last modified on   $Date:$".
+
+Part (note "Instances of part designs in the as-built state.") is a MaterialStock,
```

Comment:
MaterialStock class is not defined.

Reply:
I have changed the name of "Feedstock" to "MaterialStock" in AM-CDM_material.sadl.

This matches the main branch which was updated based on our conversations at NIST.